src/Forms/LazyLoadedLookupField.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Core\Convert;
+use SilverStripe\ORM\ArrayLib;
+use SilverStripe\ORM\DataObjectInterface;
+use SilverStripe\ORM\FieldType\DBField;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\ORM\DataList;
+
+/**
+ * Read-only complement of {@link SearchableDropdownField} and {@link SearchableMultiDropdownField}.
+ *
+ * Shows the "human value" of the SearchableDropdownField for the currently selected
+ * values.
+ */
+class LazyLoadedLookupField extends LookupField
```

Comment:
```suggestion
class SearchableLookupField extends LookupField
```
This class is used for SearchableDropdownFields, not just LazyLoaded SearchableDropdownFields

I'm think I'm happy with this extending LookupField

Reply:
I did not like LazyLoadedLookupField either. The problem I had with SearchableLookupField field is that the LookupField is not searchable. So I figured its lazy loaded, since it only loads the records that are actually needed.
But if the convention is, that we name the Lookup Field after the editable Field its for, then I guess SearchableLookupField is fine.